testing_and_setup/compass/ocean/scripts/plots/plot_initial_state.py

```diff
@@ -51,63 +52,70 @@ def main():
 
     plt.subplot(3, 3, 2)
     varName = 'maxLevelCell'
-    var = ncfile.variables[varName]
-    maxLevelCell = var[:]
-    plt.hist(var, bins=nVertLevels - 4)
+    var = ds[varName]
+    maxLevelCell = var.values - 1
```

Comment:
I always define the index variables right away to be their zero-based python variants.